07-Extract-NDVI-From-Rasters-in-R.Rmd

```diff
@@ -161,104 +166,122 @@ names(avg_NDVI_HARV) <- "meanNDVI"
 
 #view cleaned column names
 names(avg_NDVI_HARV)
-
-
 ```
 
-We are only working with one site now, however we might want to compare several
-sites worth of data. To allow us to do this, let's add a column to our `data.frame`
-called "site". We can populate this with the site name.
+However, by renaming we loose the information that this is at the Harvard
```

Comment:
i think we should introduce at the top of the lesson that we will be comparing multiple sites - so we set the learner up for this rather than saying "we might want to". If you want me to add this after the PR i'm happy to do so. OR you can add when you revise.